kube-controllers/pkg/controllers/node/ipam.go

```diff
@@ -246,15 +246,19 @@ func (c *ipamController) acceptScheduleRequests(stopCh <-chan struct{}) {
 			// Periodic IPAM sync.
 			log.Debug("Periodic IPAM sync")
 			err := c.syncIPAM()
-			if err != nil {
+			if _, ok := err.(ErrorDatastoreNotReady); ok {
```

Comment:
I don't think we need a new error branch here - the logs and behavior in each branch is effectively the same.

Reply:
Actually, see my other comments - we can probably get rid of the ErrorDatastoreNotReady type altogether.